docs/_data/sidebars/left-navigation-sidebar.yml

```diff
@@ -66,6 +66,10 @@ entries:
       url: /patterns/search-input.html
       output: web
 
+    - title: Step Input
+      url: /patterns/step-input.html
+      output: web
+
```

Comment:
How do we make the pictures for the index page?

Reply:
@estelaxu3 do you think you can help in this?

Reply:
I make them in Adobe XD :D :D :D But I am terrible at this